src/ArticlesList.jsx

```diff
@@ -0,0 +1,28 @@
+import ArticleCard from "./ArticleCard";
+import { getArticles } from "../api/api";
+import { useEffect, useState } from "react";
+
+const ArticlesList = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
```

Comment:
Nice to see you have conditionally rendered something here to tell a user the data is still being fetched while they are waiting 😄